src/Initializer/Boundary.h

```diff
@@ -23,6 +24,9 @@ struct Boundary {
     LayerMask mask = LayerMask(Ghost);
     tree.addVar(faceInformation, mask, 1, allocationModeBoundary());
   }
+
+  void registerCheckpointVariables(io::instance::checkpoint::CheckpointManager& manager,
+                                   LTSTree* tree) {}
```

Comment:
It was to provide a similar mechanism like in `LTS.h` etc. (to keep consistency between all memory layout definitions). But since we haven't implemented inheritance for them yet, I've removed them.